public/contents/5-manage-back-end-services/tours.json

```diff
@@ -12,15 +12,15 @@
       "fileSize": "21.46 KB",
       "fileName": "understandidentityservicesv9.zip",
```

Comment:
@AshokMamidiHcl Updated zip for understandidentityservicesv9 is not available in this PR. Please check.